backend/routes/classRoutes.js

```diff
@@ -1 +1,42 @@
-//here we define the routing for specific end point
\ No newline at end of file
+
+const mongoose = require('mongoose');
+const User = require('./User');
+
+
+const classSchema = new mongoose.Schema({
+            name: {
+                type: String,
+                required: true,
+            },
+            type: {
+                type: String,
+                required: true,
+            },
+            description: {
+                type: String,
+                required: true,
+            },
+            schedule: {
+                type: Date,
+                required: true,
+            },
+            price: {
```

Comment:
I think price should have a currency type and other features like hourly or full course type of plan

Reply:
don't you think that we can add that later?

Reply:
agile and all